app/main.py

```diff
@@ -1,3 +1,21 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        return (
+            days // 7
+            if days % 7 == 0
+            else
+            days // 7 + 1
+        )
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        return OnlineCourse(
```

Comment:
Use `cls` in classmethods, it's important in the case of inheritance. We'll discuss the details later on OOP topic